frame/offences/src/lib.rs

```diff
@@ -58,6 +59,14 @@ decl_storage! {
 			map hasher(twox_64_concat) ReportIdOf<T>
 			=> Option<OffenceDetails<T::AccountId, T::IdentificationTuple>>;
 
+		/// Deferred reports that have been rejected by the offence handler and need to be submitted
+		/// at a later time.
+		DeferredOffences get(deferred_reports): Vec<(
```

Comment:
Differing names isn't ideal.
```suggestion
		DeferredOffences get(deferred_offences): Vec<(
```